pytorch_vision_once_for_all.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+title: "Once for All: Train One Network and Specialize it for Efficient Deployment"
+category: researchers
+summary: Once-for-all (OFA) decouples training and search, and achieves efficient inference across various edge devices and resource constraints.
+image: https://github.com/mit-han-lab/once-for-all/raw/master/figures/overview.png
```

Comment:
This should be a logo displayed on hub's main page
have a look here
https://deploy-preview-293--pytorch-hub-preview.netlify.app/hub/research-models

Put it in torchhub `image` folder and just put the name of the logo (have a look at how [gpunet](https://github.com/pytorch/hub/blob/master/nvidia_deeplearningexamples_gpunet.md) does it)

Reply:
Sure, moved.